src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -855,6 +855,7 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
             'as_files' => true,
             'debug' => $this->debug,
             'inline_class_loader_parameter' => \PHP_VERSION_ID >= 70000 && !$this->loadClassCache && !class_exists(ClassCollectionLoader::class, false) ? 'container.dumper.inline_class_loader' : null,
+            'build_time' => $this->getContainerBuildTime(),
```

Comment:
instead of the method, what about adding the option only when a kernel.container_build_time parameter exists?

Reply:
Like this?